WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -236,14 +236,6 @@ character_event = {
 	
 	trigger = {
 		NOT = { year = 800 } # The migration period ended around 800
-		OR = { # Possible target religion groups
-			religion_group = christian
-			religion = finnish_pagan
-			religion = germanic_pagan
-			religion = baltic_pagan
-			religion = scythian_pagan
-			religion = slavic_pagan
-		}
```

Comment:
@loup99 / @EmperorEnlil 

Okay.

See, THIS is what a Game Rule would be perfect for.

**[MIGRATION RULE]**
- HISTORIC: Migrating tribes head west and roll over Europe.
- DYNAMIC: Migrating tribes can migrate anywhere in range.

That way, players can decide how they want the game to look. I'd code it myself if I could. It's all about choice.